src/main/steps/jurisdiction/interstitial/post.ts

```diff
@@ -0,0 +1,20 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { Form } from '../../../app/form/Form';
+
+@autobind
+export class JurisdictionPostController extends PostController<AnyObject> {
```

Comment:
I'm not sure what this class does, the behaviour looks the same as the parent class

Reply:
Instead of posting the formData, it assigns the formData to req.session.userCase and posts that instead